### DISCUSSION TITLE
Layout Sample Code?

### DISCUSSION THREAD

**Discussion by penta-luqman** [primefaces/community#341] [PrimeFaces Templates] [answered]
I have purchased the Avalon layout. I am looking at the demo site [here](https://www.primefaces.org/avalon/dashboard.xhtml). 
However, I cannot find where the sample code is? Similar to how is done on the showcase page, where do I find the sample code for each Avalon component? Because if I just use the component code from the [showcase](https://www.primefaces.org/showcase/getstarted.xhtml?jfwid=eb487), it looks terrible in the avalon layout. But I don't know where to find the various styles/component sample to use for Avalon.

I'm so sorry but as a paid product, I'd expect a little better experience? At the moment it appears the showcase is better documented than the paid layouts.

**Comment by jepsar** [comment, marked as answer]
In my case (Diamond) the files were present in the "tag" folder of the theme (after unpacking).

**Reply by penta-luqman** [reply]
Found it. Thank you. I wish it wasn't called tag though. Thank you!
